Model/Product/Sku/Builder.php

```diff
@@ -153,4 +159,70 @@ public function buildImageUrl(Product $product, Store $store)
 
         return $product->getMediaConfig()->getMediaUrl($image);
     }
+
+    /**
+     * Tag the custom attributes
+     *
+     * @param Product $product
+     * @param NostoSku $nostoSku
+     * @param Store $store
+     */
+    private function loadCustomFieldsFromConfigurableAttributes(Product $product, NostoSku $nostoSku, Store $store)
```

Comment:
How about putting this into trait? It's almost (only the NostoSku / NostoProduct differs) identical with this https://github.com/Nosto/nosto-magento2/pull/201/files#diff-c7bb4cf85d1abbf4cf4b2d1cec60f45aR406